src/holodeck/util.py

```diff
@@ -169,7 +169,6 @@ def logs_path():
     Returns:
         :obj:`str`: The file path of where the logs are located
     """
-
```

Comment:
Black says that it would reformat this file, but I'm not sure what it'd want to change. Is there a way to find that?

Reply:
You can run black and give it the file name as a parameter. Alternatively, you can make black your formatted in vs code and run the format code command.